src/test/java/seedu/address/logic/commands/FeedCommandTest.java

```diff
@@ -104,7 +104,7 @@ public void execute_urlIsNotAFeed_commandFails() {
     @Test
     public void execute_urlIsNotAWebsite_commandFails() {
         String expectedMessage = String.format(MESSAGE_FAILURE_NET,
-            "java.net.UnknownHostException: this.website.does.not.exist.definitely");
+            "java.net.UnknownHostException: this.website.does.not.exist.definitely: Name or service not known");
```

Comment:
`Name or service not known` might be a unix-specific error message. This test fails in windows.

Reply:
I see. Then I shall test for string containment of UnknownHostException and the website hostname